artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/TokenReview.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.activemq.artemis.cli.commands;
+
+import org.apache.activemq.artemis.spi.core.security.jaas.kubernetes.client.KubernetesClient;
+import org.apache.activemq.artemis.spi.core.security.jaas.kubernetes.client.KubernetesClientImpl;
+import picocli.CommandLine;
+
+@CommandLine.Command(name = "token-review", description = "Perform a kubernetes token review")
+public class TokenReview extends InputAbstract {
+
+   @Override
+   public Object execute(ActionContext context) throws Exception {
+      super.execute(context);
+
+      context.out.println();
+      String kubeHost = input("--kube-host", "What is the cluster host?", "api.crc.testing");
+      String kubeport = input("--kube-port", "What is the cluster port?", "6443");
+      String tokenpath = input("--token-path", "What is the token path?", "/var/run/secrets/kubernetes.io/serviceaccount/token");
+      String capath = input("--ca-path", "What is the ca path?", "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt");
```

Comment:
I'm not entirely convinced this should be a CLI command, but if it is...

Using the Option handling would seem nicer than always requiring input. Could fall back to input for e.g the token if not specified (think some other options do that).

Deferring to the the actual KubernetesClientImpl constants for defaults and key names etc (below) rather than having  distinct literal ones here might also be preferable.

Reply:
I wouldn't consider the CLI command a must have, but I figured that it would be nice to have an easy way to determine if an openshift config is correctly working or not. I'm not against removing it if you think it shouldn't be there.

Reply:
I too am not convinced of the value of the cli bit. it is quite involved and very specific. a more general approach would be through a jaas login check etc but that would be more work. In general, access to the command line should be very restricted so at best this is some sort of dev/debug tool. 
personally I would leave it out.

Reply:
Alright, leaving it out works for me! I'll remove it now.